### PR TITLE
Add ability to show system message

### LINKED ISSUE #1415: Add field in master.ini to show a system message across all pages
Looking at commit https://github.com/Submitty/Submitty/commit/cd7da78592787620e4984660f3e3cea7efa51ad4, it seems to me that it'd be beneficial to add to master.ini a new field "system_message" or something that would output on all pages in some eye-grabbing way such that we can "toggle" it on/off whenever we potentially have some sort of issue with accounts, maintenance, etc. and doesn't require us to modify the actual code.

### PR DESCRIPTION
**Pull request by MasterOdin**
Closes #1415.

You can add a `system_message` to the `config/submitty.json` file that it will show on every page of the system. If the field is missing or blank, then it's not shown. I'm not sure where we want to surface configuration of this (do we want to add a new page on explanation of the config files on submitty.org?).

screenshot 2019-01-30 17 02 01

### CODE REVIEW

**Comment by MasterOdin** [comment]
@bmcutler, this is ready to be merged.

File: site/tests/app/models/ConfigTester.php [hunk 1]
```diff
@@ -75,8 +75,9 @@ private function createConfigFile($extra = array()) {
             "vcs_url" => "",
             "cgi_url" => "http://example.com/cgi-bin",
             "institution_name" => "RPI",
-            "username_change_text" => "Submitty welcomes individuals of all ages, backgrounds, citizenships, disabilities, sex, education, ethnicities, family statuses, genders, gender identities, geographical locations, languages, military experience, political views, races, religions, sexual orientations, socioeconomic statuses, and work experiences. In an effort to create an inclusive environment, you may specify a preferred name to be used instead of what was provided on the registration roster.",
+            "username_change_text" => "Submitty welcomes all students.",
```
Comment (bmcutler) [inline site/tests/app/models/ConfigTester.php]: Why did you change this?

Comment (MasterOdin) [reply site/tests/app/models/ConfigTester.php]: It was a super long string that's used just in unit testing and I hate lines that exceed my desired length of 88 characters and really hate lines that exceed 120 characters.

It does not affect the default one that is set on installation.

**Comment by bmcutler** [comment]
@MasterOdin 
Please add some documentation of how to use it to submitty.org, maybe on this page:
https://submitty.org/sysadmin/system_customization
We don't need a UI for this, the sysadmin can just manually edit the file for now.
Then I'll merge.
Thanks

**Comment by MasterOdin** [comment]
Done: Submitty/submitty.github.io#49
